calico-cloud/reference/resources/egressgatewaypolicy.mdx

```diff
@@ -0,0 +1,83 @@
+---
+description: API for this Calico Enterprise resource.
+---
+
+# Egress Gateway Policy
+
+An Egress Gateway Policy resource (`EgressGatewayPolicy`) represents a way to select
+different egress gateways or skip one for different destinations. 
```

Comment:
This sentence is unclear...does it mean this: 

An EgressGatewayPolicy resource (`EgressGatewayPolicy`) represents a way to specify
different egress gateways as destinations during install and upgrade. If destination/CIDR doesn't have an egress gateway, you can specify a parameter to skip it.

Reply:
@bmckercher123 prior to introduction of `EgressGatewayPolicy` it was only possible to send all traffic to one egress gateway deployments. Like this:

All traffic --> egress gateway A

With this new feature, now it is possible to send traffic to different destinations to be different egress gateways (so they can route it to the desired destination). It is also possible to specify to not send egress traffic to any egress gateway and rather route it locally. This changes the setup to something like this:

Destination 1 --> egress gateway A
Destination 2 --> egress gateway B
Destination 3 --> no egress gateway

hope this helps.